target/linux/ramips/image/mt7621.mk

```diff
@@ -443,66 +443,65 @@ define Device/netgear_ex6150
 endef
 TARGET_DEVICES += netgear_ex6150
 
-define Device/netgear_r6220
+define Device/netgear_r6xxx
   MTK_SOC := mt7621
   BLOCKSIZE := 128k
   PAGESIZE := 2048
   KERNEL_SIZE := 4096k
-  IMAGE_SIZE := 28672k
   UBINIZE_OPTS := -E 5
-  SERCOMM_HWID := AYA
-  SERCOMM_HWVER := A001
-  SERCOMM_SWVER := 0x0086
   IMAGES += factory.img kernel.bin rootfs.bin
   IMAGE/factory.img := pad-extra 2048k | append-kernel | pad-to 6144k | append-ubi | \
-	pad-to $$$$(BLOCKSIZE) | sercom-footer | pad-to 128 | zip R6220.bin | sercom-seal
+	pad-to $$$$(BLOCKSIZE) | sercom-footer | pad-to 128 | zip $$$$(DEVICE_MODEL).bin | sercom-seal
```

Comment:
I would propose to not use the `DEVICE_MODEL` here  and instead introduce a `SERCOMM_HWNAME`.

Reply:
I've actually not understood what's governing the name to choose here: Can it be chosen arbitrarily or is it restricted and just matching the DEVICE_MODEL by coincidence.

Reply:
It has to match the "expectation" of the vendor firmware which matches currectly - coincidentally - the DEVICE_MODEL.

Reply:
Okay, then SERCOMM_HWNAME is the better choice indeed.